hack/sync-ec-cli-tasks.sh

```diff
@@ -25,39 +25,50 @@ set -o pipefail
 set -o nounset
 
 EC_CLI_REPO_PATH="${1}"
+RELEASE_PREFIX="release-v"
 
-cp -r "${EC_CLI_REPO_PATH}/tasks" .
+# helper function to add tasks to a git branch
+add_tasks() {
+  local remote_branch=${1}
+  local branch=$(echo "$remote_branch" | cut -d'/' -f2)
+  pushd "${EC_CLI_REPO_PATH}" > /dev/null
+  git checkout "${branch}"
+  popd > /dev/null
+  git checkout -B "${branch}" --track "${remote_branch}"
+  cp -r "${EC_CLI_REPO_PATH}/tasks" .
+  diff="$(git diff)"
+  if [[ -z "${diff}" ]]; then
+      echo "No changes to sync"
+      exit
+  fi
 
-pushd tasks > /dev/null
+  git add tasks
+  git commit -m "sync ec-cli task definitions"
+  git push origin "${branch}"
+}
 
-images="$(grep -r -h -o -w 'quay.io/enterprise-contract/ec-cli:.*' | grep -v '@' | sort -u)"
-
-for image in $images; do
-    echo "Resolving image $image"
-    digest="$(skopeo manifest-digest <(skopeo inspect --raw "docker://${image}"))"
-    pinned_image="${image}@${digest}"
-    echo "↳ ${pinned_image}"
-    find . -type f -exec sed -i "s!${image}!${pinned_image}!g" {} +
-done
+# collect remote branches in a repo returning just the branch name
+collect_remotes() {
+  echo "$(git branch -r | grep "origin/${RELEASE_PREFIX}" | sed "s|origin/||" | tr -d ' ')"
```

Comment:
I think `--format` can be used here:

```suggestion
  git branch --remote --format '%(refname:lstrip=-1)'
```